save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -105,9 +132,13 @@ internal class OrganizationController(
      */
     @PostMapping("/{organizationName}/update")
```

Comment:
organizationName is provided but it's not used, suggestion to search organization by name and replace by provided value

Reply:
Ok, will do it